src/components/lesson/Lesson.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import LessonVideo from './LessonVideo';
+import LessonInfo from './LessonInfo';
+import LessonsPane from './LessonsPane';
```

Comment:
Because of the `index.js`, as long as you import/export your components there (which you should), you can/should condense these into one line.
```suggestion
import { LessonVideo, LessonInfo, LessonsPane } from '..';
```